### ISSUE TITLE
4.2.3. Correct - Josh Jetto

### ISSUE THREAD

**Issue by ZacharyTinker** [gryphonshafer/Quizzing-Rule-Book#109]
Current Rule: 
The quizzer's response is correct when:
•	It fulfills all requirements specific to the question type
•	It contains the information requested
•	The quizzer stays in context
•	Mispronounced names are still recognizable as the answer
If the quizmaster did not complete the reading of the question, the quizzer's answer must include the information in the remainder of the question as well as the answer. The quizzer's answer must include all unique words in both the question and the answer.

Proposed Rule:
The quizzer's response is correct when:
•	It fulfills all requirements specific to the question type
•	It contains the information requested
•	The quizzer stays in context
•	Mispronounced names are still recognizable as the answer
If the quizmaster did not complete the reading of the question, the quizzer's answer must include the information in the remainder of the question as well as the answer. 

Rationale:
There is nothing about a unique word that makes it more important or more significant than the other words in the text of Scripture, such that it should not be interchangeable on question-types that do not require a word-perfect answer. Knowing that a word is unique in the text may be competitively significant as far as enabling a quizzer to identify the verse from which a question is being asked, but it is not necessarily more theologically significant than the other words in the text. For example, the word “throw” might be a unique word in the NIV 2011 translation of the Gospel of John (this is purely hypothetical). If a quizzer gave his answer and used the word “cast” instead of “throw”, under the current rule he would be called incorrect, even though the two words mean the same thing in English and are English glosses for the same Greek word in the original text. It doesn’t make any sense on question types that do not require a word-perfect answer to have a set of words that are not more meaningful than others that must be in the answer in their exact form in order for the quizzer to be called correct. The unique word rule is just an arbitrary way to have an additional (and unnecessary) reason to call a quizzer’s answer incorrect. Even if it is an objective standard that is applied equally to all, it is still an arbitrary one that is of no spiritual value. I would even go so far as to say that such a rule has a negative spiritual value if it discourages kids from trying to answer questions (or from participating in Bible Quizzing at all) and even more so if it discourages whole churches from participating in Bible Quizzing because of the (correct) perception of unnecessary, arbitrary “nit-pickiness” built into the program.

To illustrate further the ridiculousness of the unique word rule, consider this – What if we replaced the text of the unique word rule with the following rule: 
“The quizzer's answer must include all words beginning with the letter ‘S’ in both the question and the answer.”
 This would be just as objective and arbitrary a standard as the unique word rule, and it would be easier for kids to remember which words they needed to have down exactly in the text. It wouldn’t sound as ‘official’ as the unique word rule, but it would be essentially the same rule, and it would be one that we would probably all shake our heads at, which is what I do when I think about the unique word rule.  

For the purposes of Bible Quizzing, knowing which words are unique is naturally competitively advantageous, because it enables a quizzer who hears the unique word from the quizmaster’s mouth to jump earlier and get the answer correct. There is already a natural reward built into quizzing for knowing the unique words. There is no good reason to punish quizzers for not knowing the unique words and giving an otherwise, correct answer with interchangeable words to a question that did not require a word-perfect response. 

I believe we should leave it to the judgment of our quizmasters and answer judges as to whether or not a quizzer’s answer is close enough to be called correct. Sure, that means there will be some subjectivity in the answer judging process. The unique rule does not eliminate this subjectivity from quizzing; it just introduces arbitrary objectivity to it, and it ties the hands of our quizmasters and answer judges from being able to make good judgment calls regarding the interchangeability of synonymous words and phrases with respect to unique words. Short of requiring that all question types be answered word perfectly, there will always be subjectivity in answer judging. And that’s okay. That’s why we allow challenging. And in the gray area of “close enough” or “not close enough” answers, we can have good, respectful, constructive challenges and discussions in the quizzing process.

**Comment by jswingle** [comment]
I agree completely with the argument presented here. I don't see any reason why the unique words should make any difference from a ruling standpoint (although they are certainly useful as a study tool).

From a "mission statement" perspective, we already have FTV/Quote requiring word-perfect answers, which is awesome; but word-perfect answering is not the pedagogical purpose of the INT/MA/REF/SIT question types. 

The argument that "unique words are more significant than others" doesn't quite make sense to me. There are plenty of words that appear hundreds of times in Scripture which are absolutely paramount to say correctly in an answer - words such as "faith" "righteousness" "repentance" "love", etc. whereas many unique words are actually just unique conjugations of otherwise common words, for example something like "judging." If a quizzer said "he judged" instead of "he was judging," they aren't thereby missing anything significant just because "judging" is a unique conjugation.

Of course, the unique word MAY be super significant, but that's a situational thing. (And also the responsibility of the answer judge to discern, not the rulebook.) We already have rules in place which (to paraphase) dictate that the quizzer's answer needs to be the same as the text in substance, even if not word-perfect. Let's leave it to the answer judges to make wise rulings about that.

From the standpoint of a coach who works with mostly rookie quizzers, I should also add that this rule is _extremely_ discouraging to new quizzers like you wouldn't believe, to the point where I've considered but not yet pushed for a PNW-specific rule which would do the exact same thing as this proposed issue and just remove this one sentence from the rules. There should be some INT/MA/REF/SIT questions which, at the lower levels of competition, are answerable by a quizzer who did some studying but may not be able to recall the verse word-perfect. This rule makes it so that roughly a third of the time when they try, they get it wrong for completely arbitrary reasons that aren't intrinsic to the meaning of the text.

**Comment by jswingle** [comment]
As an addendum: I'm fine with rules changing a bit based on the level of competition, and I think the rulebook should focus on the Internationals level and allow for districts to make their own adjustments where necessary. So my comments about new quizzers are not the main issue here - my main argument is that the rule seems arbitrary and I don't think it serves our mission statement.

I point this out because, if it's decided the rule SHOULD stay for Internationals competition, I would push for adding commentary to the rulebook indicating that the rule is not recommended for lower levels of competition, in line with Zach's thoughts on this issue: https://github.com/gryphonshafer/Quizzing-Rule-Book/issues/113

**Comment by scottpeterson** [comment]
One thing to keep in mind was I'm pretty sure the move to require unique words was MORE a dissatisfaction in the range between quizmasters, in how they judged if the quizzer had said enough to be counted correct, as opposed to REALLY thinking all unique words have some special value. 

(I think that's a poor motivation to require unique words). But it may indicate a lack of guidance in the rule book on what constitutes a correct answer. Or it may just indicate poor quizmastering.

**Comment by scottpeterson** [comment]
> I point this out because, if it's decided the rule SHOULD stay for Internationals competition, I would push for adding commentary to the rulebook indicating that the rule is not recommended for lower levels of competition, in line with Zach's thoughts on this issue: #113


I would anticipate problems if we are starting to indicate in the rule book (which isn't explicitly labeled at the "Internationals Rule Book" even though it is stated that districts can override/ignore anything) that some rules are "you probably wanna override this at the district level."

I think there's already tension when defining some rules and structures because this single rule book is meant to encompass all. The fact that districts CAN override doesn't change the fact that there's inertia to not override. 

I think of Q/F questions, when in my opinion, the definition/purpose of these Types at the district level is VASTLY different compared to the inter-district level.

I would rather have "Internationals-only" overrides, than have the entire rule book be "for everything including Internationals, [but districts can ignore]"

**Comment by jswingle** [comment]
Agreed on the inertia side of things for sure, and we probably wouldn't want to phrase it as "do this rule instead" but rather just offer it as an alternative rule.

**Comment by jswingle** [comment]
I have heard the argument from several different people, speaking informally and in person, that unique words are "more significant." If anyone wants to jump in and argue for that, I'd love to hear more, but it doesn't make any sense to me personally.

My guess as to this history of this rule is that some districts started requiring the unique words as a "house rule" of sorts within their own district, and then there started being confusion when districts got together as to what the rule really was. It is common for even Internationals officials to revert to what they're _used_ to doing, instead of what the rulebook actually says. The rulebook never used to require unique words to be said, but since there was continual confusion about it, it was decided that we should clarify one way or the other in the rulebook. In my opinion, the way the rulebook went was the wrong decision.

**Comment by jswingle** [comment]
Which brings up another point: if we choose to get rid of this rule, should we just delete this sentence from the rulebook, or add an explanatory sentence that says something to the effect of "Unique words are not required?" If you add too many reactionary counter-explanations in the rulebook, it quickly becomes cumbersome to read and comes across like a patchwork job instead of a unified, thoughtful whole. BUT, if you don't have a rule there, a lot of people are going to revert to requiring it anyways, and then you have the same conflict with inter-district competitions.

**Comment by scottpeterson** [comment]
It should purely be deleted. The current state of the rule book shouldn't convey history.

**Comment by jswingle** [comment]
I'm inclined to agree. That issue is better solved with inter-district communication (like we hope the rulebook change management process to facilitate!)

**Comment by jttower** [comment]
My more competitive quizzers like the unique word rule, probably because it isn't subjective, but I have found that as a QM it is heartbreaking to call a quizzer wrong on an INT/MA, etc question just because they used the wrong conjugation of the word.  Take Romans 12:8 as an MA question:
"if it is to **encourage**, then give encouragement; if it is **giving**, then give generously; if it is to lead, do it **diligently**; if it is to show mercy, do it **cheerfully**."
As quizzers answer this question, they list things off in parallel structure (to encourage, to give, to lead, to show mercy), which I think is more grammatically correct.  But according to the unique word rule, I have to call them wrong for not saying the unique word **giving.**  I think this is ridiculous, even at the IBQ level.  Unique words are helpful study tools, but are not necessarily theologically significant to remember.  It is far more important that the quizzer understands what the verse means, and using a synonymous word can demonstrate that they understand the meaning of the text.  

Getting rid of the unique word rule would also help in rule 4.2.1. Context Application:
"On all question types, a quizzer cannot automatically be ruled out of context by a single word, even if it is a unique word..." Therefore, if we strike the unique word rule here, I think that rule should also be amended to remove "even if it is a unique word,"

I think that we should decide one way or the other and just be consistent with this rule, rather than making exceptions for B division or local districts.  As frustrating as it is for a quizzer to be called incorrect for missing a unique word, I think it would be more frustrating to have variations of this rule depending on where they are quizzing.  I am in favor of getting rid of the unique word requirement in questions that are not Q/F, which already have the requirement of being word-perfect.

**Comment by JoshJetto** [comment]
I think I may have addressed this in rationale elsewhere concerning the International Rulebook, but I will mention it here as well. Though legally (that is, quizzing legally), the jurisdiction of this rulebook is the International Finals, functionally this rulebook serves as the default rulebook for all the active Districts and for those who are just starting to get involved in Bible Quizzing. Additionally, my observation has been that District Quizzing Leadership Teams have tended to err on enforcing bad rules (like the unique word rule) at the local level because they want their top quizzers to be better prepared for Internationals. This line of reasoning is not good, because a) the top quizzers will get their main preparation for the International level of competition in practice after the regular season is completed and they have qualified for the team and b) the application of certain rules currently in our International rulebook are detrimental to the vast majority of quizzers at the District Level and confusing to new coaches and teams that we want to come in to quizzing. I believe that we need to make all of the rules that make up the International rulebook sensible and good for use at all levels of Bible Quizzing. Making an Internationals Rulebook with unhelpful rules or an overly-heavy reference distribution does not benefit the International quizzers, and it hurts the vast majority of District-level quizzers because those unhelpful Internationals rules tend to bleed their way back into District quizzing.

**Comment by gryphonshafer** [comment]
> I believe that we need to make all of the rules that make up the International rulebook sensible and good for use at all levels of Bible Quizzing.

For whatever it's worth, I agree with everything in this thread, but I *super agree* with this quote from Josh. While it's possible this/our rule book can be forked and altered at the district level, it really requires a pedantic rules nerd in the district to take that on. And alas, pedantic rules nerds aren't evenly distributed across all districts. This/our rule book must be assumed to be the single, canonical rule book for at least some number of districts. Therefore, any rule that doesn't function effectively to motivate/encourage the most number of quizzers to memorize the most number of verses at the district level should be avoided.

That said, I can see the argument for having some adjustable "rules dials" to optimize for competition level, so long as these are clearly spelled out in a unified way. For example, let's say you have 2 question type distributions, one optimized for the district level and another at the IBQ level. I'm not a big fan of this, though, because I've yet to see a calculation of net positive value. I'm open to being shown the data, of course.

**Comment by jswingle** [comment]
I'm an advocate of the rules allowing for different things at different levels of competition, e.g. at Internationals I think it makes sense for the question set to be harder to compensate for the increased knowledge and skill. (Quote These Three Verses should be a thing, for example - but I'll get to that in another issue someday lol)

But this is the sort of thing which should be explicitly spelled out in the rulebook as an Internationals-only thing. I agree that the rulebook language on things such as this rule should be assumed to become the default for district-level.

**Comment by jswingle** [comment]
On behalf of the Rules Committee:

During our meeting on 03/20/2022, the Rules Committee voted 6-0 to keep this issue in discussion for now.

Strong opinions were expressed, both for and against this rule. It was the unanimous opinion of the RC that given the divisiveness over this particular rule, we specifically would like to highlight this issue for more feedback from the broader quizzing world. If possible, we would like to make a final vote the next time we meet. Since the Change Management Process is new, but we would like to move quickly on this issue, we are planning to e-mail District Coordinators specifically asking for feedback from the personnel in their district.

For more information on the rulebook Change Management Process, please visit: https://github.com/gryphonshafer/Quizzing-Rule-Book/blob/master/content/rule_book/change_management.md

For the minutes of this particular meeting, please visit: https://docs.google.com/document/d/1jDIGi5LmVHbVWUmAK0-6EhJjATlI9398zK-V8Be2zNE/

**Comment by EfishWGLD** [comment]
Hey, Elijah F from WGLD here.
These are all very fair comments with great rationales. Personally, I like the rule, simply because it has always been a challenge and is a factor of quizzing that I've always had to deal with. The idea of being super close and then not getting the question because it was a different word is just an integral part of the process for me, like an additional challenge similar to having to give the question on a CVR that is unique to quizzing. That being said, I'm at a very different stage of my Quizzing life now than most people lol. I take Griffin and Zach's words for it that it discourages younger quizzers which isn't good at all. My only slight problem is the objectivity that it would present otherwise. If I get a question wrong because I missed a word, I've got no one to blame but myself; if I know I was super close but there's an especially strict QM who says "thou" isn't close enough to "you" (especially with connotations), that'll hang with me- I'd just hurt more losing those without the 
 unique word rule than with it. But if it really is making people not want to even quiz, then I'd say send it off entirely, don't add an addendum for Internats, just because that'd add more levels of confusion I think. 

If we did go away with it, which despite my statement, I'd be mostly ok with, I think we should add that actual alternative conjugations of a word are automatically correct, no questions asked (ex: threw, thrown, throws); only leave it up to the QM if it's something like thrown vs tossed.

**Comment by penningsj** [comment]
Jeffrey Pennings from WGL here. Most of my reasoning on this issue has already been stated by others so I’ll be brief and say that I agree with the thinking that key words are not necessarily spiritually significant. I feel strongly that they should not be required to be called correct because when words are not key synonyms are accepted, and those words, which are substituted with synonyms, are often more significant than the key words themselves. Hope that makes sense, but I just remember multiple circumstances where a quizzer was given the full 30 seconds to work, but ultimately called correct because what they said was close enough, and I don’t feel like purely because a word appears only once in the material that it should count a quizzer wrong.

**Comment by jswingle** [comment]
Here is a meaty comment on the same topic from a different issue thread, which I've closed out:

https://github.com/gryphonshafer/Quizzing-Rule-Book/issues/144

**Comment by Jabberwocky415** [comment]
Okay so that’s my bad for not realizing there was already a topic open on this. But yeah I wrote up quite a bit for my defense of the rule, which Jeremy has linked to.
Basically I think that the rule is a net positive for both quizzer and officials. I’ve never felt slighted by getting a question wrong this way, but I have definitely felt slighted by officials who had to interpret an answer I gave and came to a different conclusion than I.

**Comment by TimDaigle** [comment]
I really appreciate the Rules Committee proposing this change.  Thank you!  To me unique words are called out in the Quiz book to assist a good quizzer (really all quizzers) in jumping.  As many have stated, unique words typically do not hold any theological significance - in fact, it is usually the opposite, they tend to just be more obscure words.  There is no more significance to unique words as there is to saying quizzers need to get all words that start with "J" correct because Jesus's name starts with "J".  
My biggest issue with unique words was the impact on new or fledgling quiz programs.  I come from a district with a couple of powerhouse programs and the rest struggling programs.  The struggling programs had enough issues and as leaders we need to be concerned as much for them as for making Internationals difficult for the top quizzers.  As we continue to lose churches from the quizzing program, we need to do all we can to make quizzing fun and competitive.  Adding intentionally tricky rules does not accomplish either.  Cheers to you for finally addressing this important issue!

**Comment by Jabberwocky415** [comment]
If I may add one more thing. It seems like most of the arguments regarding this rule stem from the lack of theological significance inherent in unique words. As a staunch defender of the rule as currently written, I never attributed their significance to anything theological. It’s true that we require verbatim recitation of key verses because of their spiritual significance to the book. However, I believe the verbatim requirement for unique words is for an entirely different reason. Simply put, we an objective way to measure the “correctness” of the answer to an INT question. It’s a safeguard, not a statement. 

Even if I admitted that unique words were not inherently significant in themselves (which I don’t entirely believe), I would still fight to keep this rule in place. Because it’s not about the words, it’s about encouraging the kids to _memorize_ scripture, and giving them an objective goal to set for improving and sticking with their study plan.

**Comment by jswingle** [comment]
@Jabberwocky415 It's not just the lack of theological significance, it's much more importantly the lack of biblical significance. Quizzing does have the side-effect of theological/doctrinal training, but primarily we're trying to impart knowledge of the content of Scripture and the language used by the biblical authors. And when it comes to this mission, unique words are much more often than not _less important_ than non-unique words. It's the words that an author of Scripture uses _more often_ which matter _more_. (Specific examples of this later)

We partially accomplish this mission by encouraging word-perfect memorization of Scripture... emphasis on the word _partially_. We already have roughly 1/3 of the questions in a quiz exclusively devoted to word-perfect quoting, and we can add to that 1/3 this: half the time reference questions more or less require word-perfect quoting as well. I'm a huge fan of encouraging word-perfect-memorization, and I think one of the strengths of quizzing is the precision we instill in youth when it comes to handling Scripture. FTVs and Quotes should by no means change.

But we have to balance an emphasis on precision with making quizzing accessible and fun to the majority of youth. Humans are imperfect, particularly youth. It's not just reasonable, but an absolute necessity for our mission, to have some question types which don't require word-perfect precision. The unique-word-required rule takes question types which are intended to allow for a small amount of imprecision, and infuses arbitrary requirements for precision in places that, from the quizzer's perspective, are more or less random. (Yes, some quizzers know the entire key word list. That's not a relevant point when we're talking about the quizzer who memorizes some 50-100 verses imperfectly over the course of the year and just wants to jump and get a question right occasionally.)

God has created people with an absolutely massively wide range of _innate skill_ when it comes to the ability to memorize word-perfectly. I would say roughly 25% of the quizzers I've worked with literally _mentally almost cannot memorize things_. And in many cases, these are wicked brilliant, high-IQ kids. They can read all the words with high proficiency and understand them well for their age. For whatever reason, some people just can't memorize word-perfectly. I have never had this experience. I can memorize an entire chapter of Scripture faster than some kids I've worked with can memorize one single verse. So can most if not all of the people commenting on this thread - but we have to keep in mind that that's unusual. We're the nerds who really love quizzing and were/are really good at it. We have to allow quizzing to be a big enough tent to give chances for kids who are interacting with the material, but not the best at memorizing it. The current ruleset does a good job at this, in my estimation - with the exception of this one rule.

--

OK, so back to my point that the unique words are typically _less important_ than the other words, because this is a crucial part of the debate. A hefty majority of unique words are not significant at all to the thought of the author of Scripture. Some common categories of unique words include:

1. Different conjugations of less common verbs

- James 4:4 "chooses" 
- Romans 9:22 "choosing" 
- James 1:18 "chose"
- However, "chosen" appears in 4 different verses. Is this word any less important than the other three? Contextually, I would argue these occurrences and Rom9:22/Jas1:18 are way more important than James 4:4, because they concern God's election ("choosing") of his people, whereas James 4:4 is just about whether we make a choice to be a friend of the world or not. If it were not for this rule I would accept a variety of different words for James 4:4, such as "whoever _decides_ to be a friend of the world."

2. Plural vs. singular nouns in places where it doesn't matter, especially metaphors and parables

- "the autumn and spring rains" in James 5:7. "Rains" is a unique word, but there's zero reason we shouldn't count this right if the quizzer says "rain" - it grammatically is an identical statement to what the NIV says. The quizzer knows the answer, it's just not word-perfect.
- "When we put bits into the mouths of horses to make them obey us, we can turn the whole animal" in James 3:3. "Bits" and "horses" are both unique words. Zero meaning is changed if a quizzer answers an INT by saying, "When we put _a bit_ into the mouth of _a horse_ to make _it_ obey us, we can turn the whole animal." The point of the metaphor in the context is not even remotely changed - the quizzer knows the answer, just not word-perfect.
- Matthew 21:41 "he will rent the vineyard to other tenants, who will give him his share of the crop at harvest time" - the unique word is "crop." Are we really intent on calling a quizzer wrong because they say "crops"?

3. Various prefixes and contractions which can be answered equally well when de-contracted or separated as several words (all real examples of unique words):
- "unlawful" can be answered as "not lawful"
- "can't" can be answered as "cannot"
- "I'm" can be answered as "I am"
- "till" can be answered as "until"
- "won't" can be answered as "will not"

4. A whole host of other random words which are uncommon due to the methodology of the NIV translators but are not important to the meaning of the text - "whenever", "wherever", "likewise", "men", "thing"
- That last word, "thing", is a particularly illustrative example. It comes from Romans 4:6 - "David says the same thing when he speaks of the blessedness..." in this sentence, "thing" is a unique word, but "blessedness" is not. Obviously, we would require the quizzer to say the word "blessedness" because of how specific and significant it is. Are we really going to demand that the word _thing_ is the important word in this sentence?

Of course, there are many examples of highly significant unique words as well. The point is not that there aren't tons of significant ones, the point is that a substantial percentage of them are not significant, and also that there are tons of non-unique words which ARE significant. The uniqueness of a word is nearly unrelated to its significance for quizzing purposes, which is why the rule should be what it had always been before 2018: quizmasters and answer judges need to determine whether what the quizzer said is close enough to the text to be counted correct or not. It doesn't need to be objective. The QM/AJ have to adjudicate the gray area to the best of their ability, under the checks and balances of the challenging and protesting process.

**Comment by josiah-leinbach** [comment]
From William Strasburg (through the Rulebook Committee email):

"As a former quizzer and now district quizmaster, I always struggled with the unique word rule that was put into effect. 

Given how English and how many different translations of the Bible there are, it makes no sense to me how some words should be treated as more important than others. Simply appearing once in the material is certainly something to look out for to help with study and memorization, but knowing the unique words is not necessarily knowing God's word better, but the translation better. For other than quotes/ftv, which demand a particular dedication to memorization of all the words, I believe unique words should not be required to be memorized for other questions. I always hate having to call a quizzer wrong because of this rule; it doesn't feel in the spirit of quizzing.

It's a rule I have particularly despised ever since it was put into effect, as a quizzer and quizmaster."

**Comment by GeorgiaPhipps** [comment]
I remember when this rule was made and it made no sense to me then and am happy to hear it may be on the chopping block. For quizzers that have grown up in Christian families, gone to church, and been involved in programs like AWANA, they have probably already memorized some of these verses in other translations of the Bible.  Therefore they may have a synonym stuck in their brain.  If for non Q/FTV questions we allow synonyms for other words why wouldn’t we for unique words which are often insignificant words. 

Unique words certainly are a good study tool to point quizzers to a place in scripture but shouldn’t be elevated in importance above any other words, especially those which will help lead quizzers to Christ and help them grow in their faith. 

I think, even at the International level, we should do all we can to reward quizzers for studying God’s word. They put considerable time and effort in all year and then more in preparation for Internationals.  Why nitpick over Unique words. We trust Answer judges and Quizmasters to make good decisions on everything else why wouldn’t we here?

I never liked on the local level saying synonyms  were okay because it didn’t make sense to the kids that unique words were then required at Internationals.  But I refused to discourage new quizzers just to better prepare top quizzers. Although those with aspirations usually got unique words right I couldn’t say wrong to them when they didn’t. Even though the rule book is for Internationals it makes sense to have most rules that work at any level.

**Comment by theaspect09** [comment]
Hey, Emma C. from the WGLD here. I hear both sides of the debate on this rule change. Theres part of me that believes that the Scriptures should be said as word perfect, and another part of me that believes the grammar rules are a bit unreasonable. My stance on the rule change is that, so long as the message of the passage is there, keywords shouldn't be required. But! I do believe that if the keyword is pertinent to the text, then the quizzer shouldn't be able to change it, even if they learned it in a different version. We are given a certain text to memorize and take to heart, I dont think we should be able to switch from NIV to KJV or ESV whenever we memorized a quote in a different version. But I think certain exceptions can be made (the switch from 'I am' to 'I'm', 'cannot' to 'can't' and so on and so forth). Just nothing that changes the meaning of the text, because 'I'm' and 'I am' are the same word, ones just a shortening. I dont think 'transgression' should be able to change to 'sin' in an answer, even though the 'jist' is technically the same. Its still not the word the author used and I dont believe changing the words in the Bible that the author wrote is right. Thats all I got!

**Comment by kclimenhaga** [comment]
Reasoning from my district:

- Unique words and unique phrases are part of the challenge of answering INT and MA's. They may not be the most important words but they do set the verse, question and answer apart from any other question and verse, thus making them necessary
- It needs to be there for internationals as it seeks to test the depth of a quizzer's knowledge.  I think it is logical for districts to change it if they feel they must but people will put in as much effort as is required.  If we make quizzing "easier", quizzers will study to that level.  It is human nature.

Several other people disagreed with the change but did not elaborate on reasoning. Basically, WCD understands that this rule may not be in the best interest of newer quizzers, and would have no problem with the rule becoming Internationals-specific. We would like to see the rule stay for the Internationals level.

**Comment by jttower** [comment]
Personally I think it would be better to get rid of the rule entirely,  but what if we met halfway and said that some of the types of examples Jeremy gave above could be exceptions to the unique word rule?  Conjugations, plural/singular, contractions, etc.  I think it might make things more confusing, but I'm wondering where the "pro-unique word rule" people fall on this.

**Comment by taniswilder** [comment]
The rule can aid the quiz master in calling the question correct or not. Even when an alternative word is used for a non-unique word, quizzers challenge it or the quiz master and answer judge may struggle with calling the answer correct or not. 
I would like to see the rule stay. It is an incentive to have quizzers memorize and be more accurate in their memorizing. I don't think this one rule makes quizzing less fun, nor would it be a reason why some churches choose not to quiz.

**Comment by josiah-leinbach** [comment]
I think it could be helpful to conduct a formal (or even informal) poll covering most of the districts/zones to figure out how many of them, or more specifically how many of their quizmasters, enforce the Unique Word rule and to what degree. Many may dislike the rule but choose not to enforce it on either the district, zone, or individual quizmaster level. If there is a divergence between theory and practice, or at the very least a great variance in how the rule is already applied, then the solution might lie not in a rule change _per se_ but better communication to the districts about their rights and prerogatives.

I will note that this is not an endorsement of either the remove or retain position, but rather an observation that could clarify some of the underlying issues.

**Comment by Claudinecampbell** [comment]
I am all for a poll and I have an answer from Northeastern District, just not sure this is where you want it sent?  

 

Blessings,

Claudine

**Comment by jndmpeters** [comment]
Hello rulebook committee,

Thank you for the work you do behind the scenes to make Bible Quizzing what it is.

I believe that the rulebook statement requiring the quizzer to say any/all unique words should stay.  I do not think that giving quizzers larger parameters for being called correct will benefit the quizzer or the quizzing program.  I concede that it is hard for quizzer to be called incorrect for making a very small error, no matter the question type but I do not believe that changing this rule will be helpful.  Making entry into a program easier or requiring less of participants is not known to increase participation.  I worry that the rules would continue to degrade because yes, quizzing is hard, memorizing is hard and it does take work - hard work, and dedication.  But I believe that is exactly what we as Christians should be encouraging in others - the time it takes to grow closer to Christ.  If those quizzers would need to spend 15 more minutes a day to know their material better I believe it could only benefit them.  Yes, I have personal experience with quizzers that have struggled very hard with memorization and wanted to quit but kept on and were rewarded for their diligence in the long run.  Christianity is not a sprint but a marathon.  The argument could be made that many will not want to begin if it is too hard - yes, I believe Jesus said something similar, but that's exactly why I think requiring more is Biblical.

From a practical standpoint I also believe that it is short-changing them in their quizzing.  If they are called correct they will memorize that answer incorrectly and another quiz master may call them incorrect later.  Or, they may try an FTV with that same material in mind and be incorrect.  And everyone else in the room also has those incorrect words written into their minds as well.  We truly do need to encourage our quizzers.  For weaker groups it may be more labor intensive for the coaches to work at memorization with their groups instead of just quizzing and I have heard of amazing efforts and methods that some coaches have employed to do just that.

It could also slow the quizzes down quite a bit with more time taken to deliberate over whether an answer is correct and time taken to hear challenges.

I could also contest that in other areas of life "good enough" - isn't.  A song isn't correct if only the most important notes are played.  A team isn't strong if only the most important skills are mastered.  If we remember 80% of what our boss tells us to do but guess on the 20%, it might work out, but it might not.

In the end we truly need to encourage, cheer for, build up and support our quizzers.  Wrong answers may be the quickest way to learn a verse after all.

Janette Peters
Canadian Midwest Quizzing

**Comment by LanaHaight** [comment]
Unique word rule in quizzing rule book...Canadian Midwest District's official response

The issue of words being or not being spiritually/theologically significant is a moot point in Bible quizzing for these reasons.

Significance in the context of the quizzing game
• Interrogative and multiple answer questions “must include a unique word or a two- to three-word unique phase within the first five words of the question.” (Rule book 1.2.2.1)
• The person who created Bible quizzing as a game was brilliant in his/her understanding of language and how to use it to develop question types that point to one specific verse in the Scripture being studied.
• Without unique words and unique phrases, we don’t have interrogative and multiple answer questions. They are absolutely essential and significant in the context of the game.
• All games have rules to ensure consistency and fairness.

Opinion of the quiz master and answer judge
• The proposed rule says, “The quizzer's response is correct when: It fulfills all requirements specific to the question type; It contains the information requested…If the quizmaster did not complete the reading of the question, the quizzer's answer must include the information in the remainder of the question as well as the answer.”
• This proposed rule will result in quiz masters and answer judges having to determine when the information provided by the quizzer is close enough to the information in the question and answer. There is no way of guaranteeing that the QM/AJ in one room will make the same ruling as the QM/AJ in another room.
• We need to move toward more consistency in rulings and not less consistency. Less consistency opens the door to more challenges and protests. It puts the QM/AJ in difficult situations and it doesn’t allow for a higher degree of fairness for the quizzers.
• Missing a unique word is a clear violation of the current rule.

Consistency across question types
• To allow interrogative and multiple answer questions/answers to be paraphrases of the Scripture lowers the bar in terms of mastery of the material and results in more inconsistency in rulings across the various question types. We require quote/FTV/Ref questions/answers to be word perfect. Verses often have words that are not spiritually/theologically significant but we call Q/FTV/Ref quizzers incorrect if they add or omit words such as “and.” If a Q/FTV/Ref quizzer uses a contraction instead of the two words, they are incorrect. Yet, we are prepared to accept either from INT/MA quizzers even if the contraction or words are unique. This is unfair and inconsistent.

Canadian Midwest District Bible Quizzing supports calling quizzers correct as often as possible. Officials in our district look for every opportunity to do this as our goal for quiz meets is to encourage quizzers to continue memorizing God’s Word. One way to encourage students to continue Scripture memory is to have a successful meet by answering questions correctly. The best way to ensure that is to memorize verses in a word-perfect manner regardless of the type of question a quizzer plans to answer.

It’s interesting that this discussion has included comments that the rule book is for the internationals competition. The CQLT's Policy for Alliance Bible Quizzing says the leadership team’s responsibilities include: “…developing and maintaining uniform policies, including the choice of translation, facilitating the provision of resources to strengthen quizzing, including coordinating question writing, editing, distributing the same.” “Planning and giving oversight to the International Quizzing Competition” is one of nine bullet points listed under responsibilities.

First and foremost, the CQLT is to provide leadership and direction that results in a coordinated approach to Bible quizzing within the Alliance. The focus of the CQLT was not intended to be on the internationals event nor was it to prepare a rule book that applies only at the international level. We appreciate the option for districts to make amendments that apply at the district level. However, to lower the bar for INT/MA questions at the international level and to say that districts can raise the bar locally will result in significant discrepancies and is not providing leadership that promotes a coordinated approach. While not the best option, but if important to some, districts can make an amendment not requiring the unique word while the rule book continues to uphold the need for all unique words.

Canadian Midwest District Bible Quizzing supports the unique-word rule that “the quizzer's answer must include all unique words in both the question and the answer,” as stated in the rule book and it opposes the proposed rule change that would remove this requirement.

Lana Haight
On behalf of Canadian Midwest District Bible Quizzing executive committee

**Comment by jswingle** [comment]
@LanaHaight if I understand correctly, you're proposing that ALL question types require total word perfection in order to be counted correct? This is the first time I've heard this proposal, but it's also a broader discussion than just the unique-word rule. We could open a totally new issue to discuss that idea, and link to that issue in this discussion.

You mentioned that Finish and Quote questions require word perfection, but also Reference questions. Can you elaborate on this? The current rulebook never says that Reference questions have to be answered word-perfectly, to my knowledge. Reference questions end up being more persnickety than INT/MA questions just because the quizzer can't say an answer that comes from a different chapter or verse, but word perfection is not a requirement anywhere in the rulebook. If you'd like to propose it become a requirement, though -- I frankly wouldn't have a problem with that. I just wouldn't want it for INT/MA questions.

**Comment by LanaHaight** [comment]
@jswingle 
Canadian Midwest is not proposing that all question types require word perfection. As for reference questions, quiz masters in our district don't ask for the question until all the information has been given. If a quizzing doesn't have the wording correct, quiz masters won't ask for the question. When it comes to providing the question, the quizzer would need to given correct wording there as well.

**Comment by ARMediting** [comment]
I'd like to note that that C&MA quizzing as a program seems to be trending toward greater reliance on unique words and phrases over the years. It's now standard to have unique words bolded in quiz books. An INT question now must have a 1-, 2-, or 3-word unique phrase in the first 5 words. Concordances are listing 2- and 3-word phrases now. I'm curious if there is a reasoning behind this. Is it simply because computers make collecting them easier? Is it the nature of increased competition? Do we want to encourage studying these things?

If there's a desire for less ambiguity in rulings, why not match 1.2.2.1's requirements for a 1-, 2-, or 3-word unique phrase to be given word-perfect by the quizzer? I'm against this personally, because it wildly expands the amount of insignificant phrases required for correctness, but it seems to be the logical continuation of the rule. (I'd like to revisit 1.2.2.1. as well, but that's for another issue.)

Another solution that seems like more of a compromise between sides is creating a "significant" words/phrases list that doesn't change from year to year. This could be a published list, highlighted phrases in books, or a description of what kinds of topics go on it. (Acme's 4-in-1 highlighted Scripture [https://www.acmequiz.com/images/products/2021_4n1.pdf](url) offers one model.) This would eliminate the concern of non-significance but keep a system of more objectivity.

**Comment by ARMediting** [comment]
Actually, giving it some more thought, wouldn't the deity (Trinity) rule be our attempt to enforce a type of perfection on significant phrases? It could be expanded, maybe to all proper names, or strengthened, to require the exact phrase used in the verse instead of just the right member of the Trinity.

**Comment by jswingle** [comment]
@ARMediting I'm not sure the deity/Trinity rule is exactly the same category as this rule. The Trinity rule, at least how I've seen it, is our attempt to (a) apply proper Trinitarian theology to the rule about the quizzer not being allowed to give incorrect information, (b) enforce the requirement that quizzers must give the information required, by requiring them to say the title used in the text and not just any old title for that person of the Trinity.

So the Trinity rule is just an application of our most basic rules regarding answering questions: (a) that the quizzer can't give incorrect information, and (b) that the quizzer must eventually say the information required to be counted correct.

I would describe the Trinity rule as emphasizing _materially correct_ answers, not _word-perfect_ answers.

**Comment by JoshJetto** [comment]
Responding to @ARMediting from June 4...In one of my proposals to the CQLT back in 2017, I included this option of a significant word list. However, the problem I see with it is that - with the exception of proper names and titles - there are actually good synonyms for just about any Scripturally significant word in English, especially when you see across the English translations of the Scriptures different, synonymous words being used to translate the Greek text - for example "righteousness - justification - justice," "holiness - sanctification," "atoning sacrifice - propitiation," - as well as in situations where the Greek text uses synonyms for parallelism that become reflected in different English synonyms - "Blessed are those whose _transgressions_ are _**forgiven**_, whose _sins_ are _**covered**_. Blessed is the one whose sin the Lord will never count against them."  In other words, even for significant words, there are good synonyms, and a quizzer who uses the synonym instead of the word that the translators chose may actually be reflecting a better understanding of what the text is saying than someone who has it down word-perfect according to the English text. A "significant words" rule is better than a "unique word" rule, but it still carries with it a certain arbitrariness because there are almost always good synonymous words and phrases that can be substituted and that reflect a correct understanding of the text even if not a verbatim repetition of it. I am much more in favor of trusting our quizmasters (and answer judges) to make right judgment calls with the checks and balances of challenges and protests provided in the rulebook than with tying the hands of the quizmasters with rules that don't allow them to make those good judgment calls regarding the interchangeability of synonymous words and phrases. In fact, I would suggest that allowing quizmasters and answer judges to make these judgment calls actually spurs the captains and co-captains to understand the words and the meaning of the text better in order to be able to make challenges that concern the nuances of what might or might not be a close-enough wording to convey the correct meaning of the Scriptures.

**Comment by jswingle** [comment]
Couldn't agree more with @JoshJetto 's reasoning. Trusting the process of answer judging / challenging / protesting and preparing our captains to make a good challenge is the key.

**Comment by ARMediting** [comment]
My question about the Trinity rule was more of an observation that there are already certain places in INTs that we all generally agree should be given to a greater degree of accuracy than the surrounding material. (Similarly, in some Gospel years, the Internationals officials often clarify which groups of religious leaders are synonymous and which should be given exactly. Sure, it's more of a clarification than a strengthening of a rule, but the effect is to reduce ambiguity and prevent needless challenging.)

The existence of these two side cases, as well as Josh's point on July 5 (which I don't totally agree with, however) is why I suggest proper names and titles as a type that quizzers would need to give perfectly.

My overall argument, though, is that since both sides feel very strongly about this issue, perhaps a compromise is better than one side or the other getting all of what they want. I don't want this to turn into a political thing where people try to get on the Rules Committee just to change it back and forth. I don't really want a different set of words to have to enforce, but is it better than what we've got? Yes.

**Comment by jttower** [comment]
There's clearly strong feelings on both sides here, so I still feel a compromise may be necessary.  What if we met halfway and said that some things could be exceptions to the unique word rule? Conjugations, plural/singular, contractions, etc. I would really like to hear where the "pro-unique word rule" people fall on this as it doesn't make sense to me to call a quizzer wrong if they are literally one letter off (choose/chose/chooses, they used a contraction, etc). See the comment by @jswingle on Mar 30 for examples of this.  I would also like to hear from the "anti-unique word rule" people to see if this could perhaps be a workable compromise.  I agree with Alex that we don't want this rule flip-flopping based on who is on the Rules Committee, but I know we would also like to progress this issue.

**Comment by EfishWGLD** [comment]
I like the idea of accepting words that are very slightly different (i.e. choose, chose), contractions (i.e. can't, can not), and plurals (i.e. human, humans) for key words over a more difficult, larger scale change of accepting close substitutes for words that are key (i.e. tossed, threw). All of those three slight variations in the words convey the same connotations of the words in the text and would thus not change the overall meaning of the verse besides a minor grammatical issue (which are already present in certain areas according to convention anyway). This would also clear up the issue better with quizmaster judgement and quizzer challenges. Especially in non-quoting questions like interrogatives, I think it makes sense to allow for (basically) exact substitutes for words, and in my experience with answering questions, I believe it would make more sense to give a quizzer points for the correct word (minus an "s" or something not changing the word's connotation) if they have given it. This would solve the issue of a quizmaster being limited by the rule from calling someone correct who is correct in every sense except for the broad Unique Word Rule. It's a solid compromise for those of us who don't like the idea of allowing a quizzer to be correct without the right word that though similar, is used in different circumstances, actually concretely means something different, etc.

**Comment by LukeBraisted** [comment]
I personally think it would be best to get rid of the unique word rule entirely. It draws attention to words that are not necessarily more significant in meaning, and there is a possibility that the requirement could discourage new quizzers.

However, I would also be open to the idea of keeping the rule but adding exceptions for different forms of the same word (conjugations, prefixes and suffixes, plural vs. singular, and contractions). This would be a compromise that would keep more objectivity and precision while also preventing much of the confusion and frustration that could occur because of word forms that are nearly identical in sound and meaning.

**Comment by JoshJetto** [comment]
Prior to the 2017 rulebook update, the unique word rule did not exist in any of the official rulebooks we still have a record of (going back to 2002). Going back to 1994 (when I started as a quizzer), it was not to my knowledge a written rule either. Before 2017, the unique word rule was a District house rule for some Districts and not a part of the International rulebook (aka the standard/default/coordinated rulebook for all the Districts). So, for at least the 23 years of C&MA Bible Quizzing prior to 2017 (and I would surmise further back than that), the unique word rule was not enforced at Internationals but was an add-on that some Districts chose to use. Not having this house-rule make its way to the International competition did not create problematic discrepancies in the competition nor was it a detraction from C&MA quizzing at the International level nor did it intrude upon Districts who were using the official, standard rulebook in those years from which the unique word rule was absent. 
However, in 2017, the unique word rule was added to our standard rulebook, and as a result, it became the default rule for all the Districts because it became the rule for Internationals. Prior to the 2017 change, some Districts enforced the unique word rule at their District competitions and then came to Internationals where it was not enforced (and quizzed just fine without it), and other Districts who were either unaware of this house rule or saw no benefit to it, quizzed without it at their District meets and at Internationals (and quizzed just fine without it). The addition of the unique word rule in 2017 to the International (standard) rulebook put pressure on Districts that were not previously using this as a rule to begin doing so - and as has been expressed above, doing so has not been positive change for a significant portion of the quizzing community. The point is this: the unique word rule can be removed from the Internationals rulebook without changing anything for those Districts that wish to use it at the District level. However, the presence of the unique word rule in the Internationals rulebook does change District level quizzing in Districts that would not otherwise use it (because they believe it to be a poor rule), because its presence in the official Internationals rulebook for C&MA Bible Quizzing puts pressure on all the Districts to enforce this rule at their own District level, simply because it is currently the rule at Internationals.

**Comment by JoshJetto** [comment]
I won't rehash the reasoning I gave in the initial post on this thread for why the unique word rule is both illogical and detrimental to quizzing (I had sent it to Zachary Tinker who posted it on my behalf before I learned how to use GitHub - if you haven't read it, I would encourage you to do so). I will, however, add an additional reason why I am in opposition to the unique word rule here.  

First, the Unique Word Rule creates “out-of-order” study priorities that work against the kind of Scripture memorization and study that contributes best to the Mission of Bible Quizzing.
As a coaching priority, my study priorities for Bible Quizzers are, in this order:
1) Read all of the quizzing material over, out loud, repeatedly, so that they become very familiar with the text. Doing this also helps them to see the individual verses in their overall context, to understand the flow of the stories or of the teaching being conveyed.
2) Memorize the Club 150 or Club 300 verses
3) Memorize all of the Verses
4) Then, if they are doing #s 1 and 3, and only after they are doing these two things, would I recommend studying the unique word list. (In fact, I don’t remember if I even began to study the unique word list until after I had made the International Team; it was not as important as memorizing and reading). In other words, knowing the unique word list should only be a study priority for Full Material quizzers.

According to the records kept by the Canadian Midwest District, roughly 10% of the quizzers in their program - and I would guess that this roughly holds true across all the Districts. Another 10-15% on average complete Club 300, and then a whopping 40% complete Club 150 each quiz season. (I think this is great, by the way, and have shared these stats in our District in promoting the value of Bible Quizzing as a vehicle for motivating young people to know what God's word says). 
As it pertains to the unique word rule, however, if only 10% of quizzers are memorizing Full Material, and 10% are memorizing Club 300, this means that 80% of the quizzers at the District level will only be at a paraphrase level of knowledge of the vast majority of the quizzing material each year. The presence of the unique word rule further disadvantages these quizzers when competing with Club 300 and Full Material quizzers, because it makes their paraphrase knowledge of the text not good enough on questions that do not require a word-perfect answer. This is not a benefit to quizzing or quizzers. It makes the competition discouraging for lower club quizzers since it increases the likelihood they will err on verses outside of their clubs, and it makes the competition easier for Full Material quizzers, since more of the lower club quizzers will err on questions they would otherwise answer correctly were it not for the unique word rule. And when the external competition is lower, so is the motivation to know the material better, because you don't have to be as sharp to still quiz out in every quiz.

The Unique Word rule places undue emphasis on knowing single words that are of no greater significance in terms of meaning than any other words in the text. And in terms of studying, it puts greater weight on learning random verses or phrases in the text rather than on studying the whole text. At best, it elevates learning the unique words to a priority above memorizing and below reading. At worst, it causes quizzers to feel the need to learn these arbitrarily-selected words above the need to take in and grasp the whole of the book or section being studied, which does not help them to grasp the truth of what God is saying in His word and thus to believe and obey it.

**Comment by GeorgiaPhipps** [comment]
Amen!  I wholeheartedly agree with your letters. Your ending says it all. Let’s not loose sight of what is truly important. We can help a quizzer become a top quizzer, follow all the guidelines, know the unique words, hold trophies, but if they don’t know Christ, understand, believe, live and act on his word we have failed to guide them toward what has eternal consequences. At some point most toss the trophies but prayerfully continue to live and serve a living Savior.

**Comment by josiah-leinbach** [comment]
Removing the "unique word rule" simply federates the issue to the district level. If it makes sense for a district to enforce it or not depends entirely on the district's circumstances and whether it would be of benefit. Districts already employ rules or practices that are not in keeping with the International rulebook, and that is fine. And in the grand scheme of rules, the unique word rule is not nearly as significant a rule difference from district to district as some of those that already exist (e.g. team vs. individual bonuses, question distribution, etc.).

**Comment by josiah-leinbach** [comment]
I would add that one of the main problems here is that the purported harms of the rule are quantitative and obvious (e.g. quizzers getting questions wrong because they missed a unique word), whereas the purported benefits are not quantitative and not obvious (e.g. raising the caliber of quizzing). That does not mean the purported benefits do not exist, but that such asymmetry makes it next to impossible to evaluate whether the overall effects of the rule are good or not. 

But I would argue that the one of the main purported benefits of the unique word rule — which, at least so far as I have discerned, is that it raises the caliber of quizzing — is not actually that intuitive of an argument. It could just as well be argued that the way you raise the caliber of quizzing is to develop a "middle tier" of quizzers, the kind most harmed by the unique word rule. These quizzers are the kind who rarely quiz out but still jump on a 2-3 questions a quiz and get half of them correct. The more of these quizzers that are in a given quiz, the fewer questions Internationals-caliber quizzers (i.e. those likely to quiz out) can jump on. Because these "middle tier" quizzers do not memorize full material but still know it well enough to jump on INT questions with decent accuracy, the unique word rule as it stands discourages them from jumping or penalizes them if they do. Thus, there are fewer middle tier quizzes, which increases the available questions for the Internationals-caliber quizzers, thereby making it easier for them.

Far from lowering the standards of quizzing, removing the unique word rule makes for more competition at the level where it is needed the most. Quizzers who memorize full material will still quote things word-perfectly or close to it. Few if any of them will cease to study the unique words, because they still have great practical value for jump points.

**Comment by 24RMiller** [comment]
My coach(Andrea) keeps encouraging me to give my opinion on this topic, so I guess I will give it a try.

I will start by saying that I see the value in the unique word rule. It requires quizzers to know the material better, and it makes it easier for quizmasters to judge. However, I still feel that it takes quizzing to a unnecessary level of perfection. Already in a quiz we have 5 or 6 memory questions. Some quizzers are built for those types of questions; others aren’t. In my opinion, the unique word rule has forced everyone to know the specific wording of the material better, which in and of itself is not a terrible thing.

Despite this, I still find major flaws with the unique word rule. In many cases, unique words are no more “spiritual” than other words. The only thing that makes them “more significant” is the fact that they only appear once in the material. My question then is: why are words that appear once required, but words that appear twice aren’t? Why aren’t unique 2 word phrases required? What actually makes unique words that special? 

In my studying today in Acts chapter 7, I saw multiple examples of issues with this rule:

1. In Acts 7:13, “Joseph’s” in the phrase “Joseph’s family” is a unique word. This means that if a quizzer answered “the family of Joseph”, they would be counted wrong. 

2. In 7:26, “upon” is a unique word. Upon is a preposition, and many prepositions have similar, if not identical definitions. Here again, I don’t see why a quizzer should be counter wrong if they use a phrase such as “up to” instead of “upon”. Additionally, this brings up the question, “what if the quizzer says ‘up on’ as two separate words?”

Which leads to my next point: adding exceptions the rule. I believe one reason for the unique rule is to make it easier on quiz masters. While on the surface, I like the idea of having exceptions, such as plurals and different forms of the word, I am afraid that that will make it harder on the quiz master. One specific example is the word “go”. The past tense of “go” is “went”. Words like these would make it hard for a quizmaster to always determine if it is the same root word. 

A final point: the unique word rule has removed some of the subjectiveness of quizzing, and as a result, the influence of challenging. The unique word rule gives the captains less room to argue whether or not the phrase given is “close enough” to the phrase in the answer. 

All of this to say: I feel the unique word rule should be removed from the rulebook. Words should not be given special significance simply due to the fact the appear once in the material. I believe that the ultimate goal of quizzing should be for quizzers to know what God is saying in His word. Punishing a quizzer for not knowing a tense or a synonym does not seem to fit this goal of quizzing. Memory questions should require perfection, but the other 13-14 questions should be testing a quizzer knowledge of the content of the material, and not the specific wording.

**Comment by ARMediting** [comment]
While I initially liked Jessica's proposal, I started looking at the types of unique words in a few chapters of Acts. What I found was that some unique words could mean the same thing if made plural, put in the past tense, etc., but some would drastically change the meaning of the Scripture if changed in exactly the same way. 

Acts 2:6 says, "a crowd came together in **bewilderment**." If a quizzer says, "A crowd came together bewildered," the meaning has not changed.

However, 2:17 says: Your sons and **daughters** will prophesy, your young men will see **visions**, your old men will **dream dreams**. A quizzer who says "Your old men dreamed dreams" or "Your young men will see a vision" should not be counted correct; they've changed the meaning of the verse.

I'd say it's fairly easy to tell which changes affect meaning and which don't, though there would be room for challenges. That's why I'm sticking to my compromise proposal of requiring exact proper nouns. We can agree those are important, they're easy to identify even in a Bible that doesn't mark uniques, and the text and complexity of the Trinity rule could be reduced.

(Anyone interested can check out my highly unscientific breakdown of a couple of chapters of unique words: https://docs.google.com/spreadsheets/d/15PrCGfqII1DTADctEzi5qczqt73TZ2kjtD3P8PIXqpA/edit?usp=sharing)

**Comment by jswingle** [comment]
On behalf of the Rules Committee:

During our meeting on 11/06/2022, the Rules Committee voted 5-1 to approve the change detailed below. This means that a "pull request" has been made - in other words, a change has been officially proposed and voted on by the Rules Committee, and will be given a final vote in 3 months.

We voted to remove the rule which requires quizzers to say unique words to be counted correct. To be specific, the only change this makes to the rulebook is the removal of the following sentence: "The quizzer's answer must include all unique words in both the question and the answer."

A number of concerns were raised in support of removing the rule, many of which were brought up in this issue's discussion. Concerns with the arbitrariness of the rule and the difficulty of the rule to newer quizzers were brought up. Counterproposals and arguments were made to the effect that it is a positive to have objective things to rule on, and the requirement of unique words helps make rulings more objective. The desire for objective rules was agreed upon by everyone in the Rulebook Committee, but the vote was still made to remove the rule due to the other considerations. Instead, we want to seek other ways to make rulings more uniform and objective.

For more information on the rulebook Change Management Process, please visit: https://github.com/gryphonshafer/Quizzing-Rule-Book/blob/master/content/rule_book/change_management.md

To view minutes for this Rulebook Committee meeting, please visit https://github.com/gryphonshafer/Quizzing-Rule-Book/wiki/Rules-Committee-Meeting-Minutes

**Comment by TimDaigle** [comment]
Thanks for the update Jeremy.  I am thrilled that the Rules Committee took up this issue and with your proposed change!

 

Tim

 




 

 <http://linkedin.com/in/tidaigle/>   <https://www.facebook.com/groups/TheGatheringACAC>   <http://twitter.com/daiglecup> 

Tim Daigle

Marketplace Ministries

Email:  ***@***.***> ***@***.*** 

Phone: +1 412-897-6462 

250 East Ohio St, Pittsburgh, PA 15212
